pkg/virt-handler/device-manager/device_plugin_base_test.go

```diff
@@ -79,4 +82,79 @@ var _ = Describe("Device plugin base", func() {
 			Expect(fakeServer.count).To(Equal(3))
 		})
 	})
+
+	Context("device plugin lifecycle", func() {
+		var workDir string
+		var err error
```

Comment:
Please don't use shared error as it can lead to more mistakes.

Reply:
You are right, done